fuel-p2p/src/discovery/mdns.rs

```diff
@@ -1,8 +1,8 @@
 use libp2p::{
     mdns::{
-        MdnsConfig,
-        MdnsEvent,
-        TokioMdns,
+        tokio::Behaviour,
```

Comment:
I think it would more readable that it's imported as `TokioMdns`, so that we know which behaviour it is (Mdns) and which implementation it uses (Tokio). We should use that approach throught libp2p imports, for behaviours and events, it makes more sense especially when you import multiple Events/Behaviours within a single file.

Reply:
Do you mean like `tokio::Behaviour as TokioMdns`?